app/Models/News.php

```diff
@@ -101,6 +101,38 @@ public static function getNews()
     public function scopeGetPublished(){
         return News::where('published','=',1);
     }
+    
```

Comment:
Code should be something like this
// in your model file
public function next(){
    // get next user
    return User::where('id', '>', $this->id)->orderBy('id','asc')->first();

}
public  function previous(){
    // get previous  user
    return User::where('id', '<', $this->id)->orderBy('id','desc')->first();

}
// in your controller file
$user = User::find(5); 
// a clean object that can be used anywhere
$user->next();
$user->previous();

Reply:
Also model should't know about btns